api/http/management.go

```diff
@@ -207,6 +222,72 @@ func (h ManagementController) Connect(c *gin.Context) {
 	h.ConnectServeWS(ctx, conn, session, deviceChan)
 }
 
+func (h ManagementController) Playback(c *gin.Context) {
+	ctx := c.Request.Context()
+	l := log.FromContext(ctx)
+
+	idata := identity.FromContext(ctx)
+	if !idata.IsUser {
+		c.JSON(http.StatusBadRequest, gin.H{
+			"error": ErrMissingUserAuthentication.Error(),
+		})
+		return
+	}
+
+	tenantID := idata.Tenant
+	userID := idata.Subject
+	sessionID := c.Param(PlaybackSessionIDField)
+	session := &model.Session{
+		TenantID: tenantID,
+		UserID:   userID,
+		//DeviceID: deviceID,
```

Comment:
I actually removed this line, since we do not have this information at this point (it is stored in the auditlogs, but to playback only the session id is required)